From 02519ad921d3479855fe8fb9e2848e0d8b460c9d Mon Sep 17 00:00:00 2001
From: Vlad Zagorodniy <vladzzag@gmail.com>
Date: Sun, 15 Jul 2018 15:45:19 +0300
Subject: [PATCH] [kdecoration] Make shadows lighter

Summary:
#### Small

Before:
{F6107651, layout=center, size=full}

After:
{F6108127, layout=center, size=full}

#### Medium

Before:
{F6107660, layout=center, size=full}

After:
{F6108129, layout=center, size=full}

#### Large

Before:
{F6107662, layout=center, size=full}

After:
{F6108131, layout=center, size=full}

#### Very large

Before:
{F6107666, layout=center, size=full}

After:
{F6108133, layout=center, size=full}

#### With normal background

{F6108141, layout=center, size=full}

Reviewers: #vdg, ngraham

Reviewed By: #vdg, ngraham

Subscribers: plasma-devel

Tags: #plasma

Differential Revision: https://phabricator.kde.org/D14133
---
 kdecoration/breezedecoration.cpp | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/kdecoration/breezedecoration.cpp b/kdecoration/breezedecoration.cpp
index 630ce1ba..7177b3d4 100644
--- a/kdecoration/breezedecoration.cpp
+++ b/kdecoration/breezedecoration.cpp
@@ -104,24 +104,24 @@ namespace
         CompositeShadowParams(),
         // Small
         CompositeShadowParams(
-            QPoint(0, 6),
-            ShadowParams(QPoint(0, 0), 16, 0.9),
-            ShadowParams(QPoint(0, 0), 4, 0.1)),
+            QPoint(0, 4),
+            ShadowParams(QPoint(0, 0), 16, 0.6),
+            ShadowParams(QPoint(0, -2), 4, 0.14)),
         // Medium
         CompositeShadowParams(
-            QPoint(0, 12),
-            ShadowParams(QPoint(0, 0), 32, 0.94),
-            ShadowParams(QPoint(0, 0), 16, 0.1)),
+            QPoint(0, 8),
+            ShadowParams(QPoint(0, 0), 32, 0.7),
+            ShadowParams(QPoint(0, -5), 14, 0.12)),
         // Large
         CompositeShadowParams(
-            QPoint(0, 20),
-            ShadowParams(QPoint(0, 0), 64, 1.0),
-            ShadowParams(QPoint(0, 0), 24, 0.1)),
+            QPoint(0, 18),
+            ShadowParams(QPoint(0, 0), 64, 0.8),
+            ShadowParams(QPoint(0, -10), 24, 0.1)),
         // Very large
         CompositeShadowParams(
-            QPoint(0, 34),
-            ShadowParams(QPoint(0, 0), 96, 1.0),
-            ShadowParams(QPoint(0, 0), 28, 0.1))
+            QPoint(0, 26),
+            ShadowParams(QPoint(0, 0), 96, 0.95),
+            ShadowParams(QPoint(0, -12), 28, 0.1))
     };
 
     inline CompositeShadowParams lookupShadowParams(int size)
-- 
2.18.0

